Analyse_Birds.qmd

```diff
@@ -457,7 +455,268 @@ Here we have **58%** of good predictions, which is somewhat below the common thr
+```
+
+Problem : for most of the variables the chi² test can't be done because of the low number of values in each group. The only one that fulfill the conditions and therefore can be made is between Habitat and Species but the result shows that the variables are correlated.
+
+## B. Statistical Analysis
+
+Now that we have a better idea of the data implied in this study, we can begin the statistical analysis.
+
+## Building of the model
+
+Because our response variable follows a poisson distribution we use a link function : **the log function**. 
+
+```{r poisson_fullmodel,include=TRUE}
+# Model formulation
+mod1<-glm(Number_Fledged~ County
+        + Species
+        + Habitat
+        + Year
+        ,data=data_filtered
+        ,family=poisson(link="log"))
+# Variable type : poisson. Link fonction : log
+# Then we check for significance
+drop1(mod1,test="Chi")
+```
+We decide to remove the less significative variable : here it's Year.
+
+```{r poisson_model1,include=TRUE}
+# Model formulation
+mod1<-glm(Number_Fledged~ County
+        + Species
+        + Habitat
+        ,data=data_filtered
+        ,family=poisson(link="log"))
+#type de la variable : poisson et fonction de lien de type log
+# Then we check for significance
+drop1(mod1,test="Chi")
+```
+
+Now the less significative is Habitat
+
+```{r poisson_model2,include=TRUE}
+# Model formulation
+mod1<-glm(Number_Fledged~ County
+        + Species
+        ,data=data_filtered
+        ,family=poisson(link="log"))
+#type de la variable : poisson et fonction de lien de type log
+# Then we check for significance
+drop1(mod1,test="Chi")
+```
+
+County is still not significative so we have to remove it.
+
+```{r poisson_model3,include=TRUE}
+# Model formulation
+mod1<-glm(Number_Fledged~ Species
+        ,data=data_filtered
+        ,family=poisson(link="log"))
+#type de la variable : poisson et fonction de lien de type log
+# Then we check for significance
+drop1(mod1,test="Chi")
+
+summary(mod1)
+```
+
+We have the final model with only the Species.
+
+-   $Species{Bobolink }$ = 0 (intercept)\
+-   $Species{Dickcissel }$ = $-0.18^{**}$
+-   $Species{Eastern Meadowlark }$ = $-0.14^{**}$
+-   $Species{Grasshopper Sparrow }$ = $-0.05$
+-   $Species{Henslow's Sparrow }$ = $-0.18$
+-   $Species{Savannah Sparrow }$ = $-0.19^{***}$
+-   $Species{Vesper Sparrow }$ = $-0.32^{***}$
+-   $Species{Western Meadowlark }$ = $-0.04$
+
+The equation of the model is the following:
+
+$$ Number_Fledged = 1.29^{***} + (Species{Bobolink} = 0 ;\:Species{Dickcissel} = -0.18^{**} ; \:Species{Eastern Meadowlark} = -0.14^{**} ; \:Species{Grasshopper Sparrow} = -0.05 ; \:Species{Henslow's Sparrow} = -0.18 ; \:Species{Savannah Sparrow = -0.19^{***} ; \:Species{Vesper Sparrow = -0.32^{***} ; \:Species{Western Meadowlark} = -0.04) $$
+
+According to the model's summary, the null deviance is of 599.76 and residual deviance is of 572.82 in order to estimate the explained deviance, one can use the following formula:\
+
```

Comment:
in term of structure i would put a dot after 572.82 and start a new sentence with "In order"